apps/aecore/src/aec_mining.erl

```diff
@@ -18,23 +18,39 @@
 
 %% API
 
--spec create_block_candidate(aec_blocks:block(), aec_trees:trees(),
-                             list(aec_headers:header())) ->
+-spec create_micro_block_candidate(aec_blocks:block(), aec_blocks:block(), aec_trees:trees()) ->
                                     {ok, aec_blocks:block(), aec_pow:nonce()} |
                                     {error, term()}.
-create_block_candidate(TopBlock, TopBlockTrees, AdjHeaders) ->
-    create_block_candidate(get_txs_to_mine_in_pool(),
-                           TopBlock, TopBlockTrees,
-                           AdjHeaders).
+create_micro_block_candidate(TopBlock, CurrentKeyBlock, TopBlockTrees) ->
+    create_micro_block_candidate(get_txs_to_mine_in_pool(), TopBlock, CurrentKeyBlock, TopBlockTrees).
```

Comment:
If this was `created_signed_micro_block_candidate(...)`, then quite a lot of code could be removed from `aec_conductor`, such as `start_micro_signing` and `handle_micro_signing_reply`. It's already done like that for [coinbase tx](https://github.com/aeternity/epoch/blob/98db894d5d02b513e83ff1f69f2747f038590bb6/apps/aecore/src/aec_mining.erl#L31-L47).

Reply:
@jur0 please proceed with refactoring